spec/controllers/kalibro_configurations_controller_spec.rb

```diff
@@ -1,6 +1,14 @@
 require 'rails_helper'
 
 describe KalibroConfigurationsController, :type => :controller do
+  
+  def post_method(method)
+    unless method == :create
+      post method, :id => kalibro_configuration.id, :kalibro_configuration => kalibro_configuration_params, :attributes => attributes
+    else
+      post method, :kalibro_configuration => kalibro_configuration_params, :attributes => attributes
```

Comment:
It saves us some lines of code. And the action for update was tested with `post` before this modification. So the behaviour was not changed.

I'll fix the test to use `put`.